### PR TITLE
FEAT: home page, side nav and basic card

### PR DESCRIPTION
**Pull request by partha8**
The current PR features a home page with a side navigation bar. The card component here is a basic one and I will make changes to it in the upcoming PR. 

## Note
The PR is not as big as it seems. Several lines of code are repeated from previous projects and some of the CSS stylings are already reviewed.

## files to check
- [src/components/Card/Card.js](https://github.com/partha8/melonn/compare/Home?expand=1#diff-c140b1cd54cc14c0d77d8af6d1b69d4a1e5456147d4379c8fcb3629e651d0392)
- [src/components/Navigation/Navigation.js](https://github.com/partha8/melonn/compare/Home?expand=1#diff-dc21f819aa805242007e4062e2b283f116b5ffd173e44222fdf495face09cc89)
- [src/components/Navigation/navigation.module.css](https://github.com/partha8/melonn/compare/Home?expand=1#diff-40837133f09664d4cbbce13128a12c9248fe38f751881cda8af037846991632b)
- [src/context/AppProvider.js](https://github.com/partha8/melonn/compare/Home?expand=1#diff-23452557229bcbbdc31735f84a6e79ed51ff410a962efeeedcdcfd05b8726109)
- [src/context/NotesProvider.js](https://github.com/partha8/melonn/compare/Home?expand=1#diff-585c9a76075cc76bdae9dea74243ed5b98df79dcaff96225f00c61ce56d04090)
- [src/hooks/useGetNotes.js](https://github.com/partha8/melonn/compare/Home?expand=1#diff-422b3cdaae64a7e2c174844b81c45ee6500403a4ecbb243511cc61b80b66110a)
- [src/hooks/useGetPinnedNotes.js](https://github.com/partha8/melonn/compare/Home?expand=1#diff-7a6f569a3ae5b16d1032c50ff9eccda1d8312cd53e5d76a124741fbb9f3e8fd9)
- [src/pages/Home/Home.js](https://github.com/partha8/melonn/compare/Home?expand=1#diff-f4fa90c831fad21520938358d61dd9b568559f33560f768e17cc3263c78ee6a0)

## The preview
image

### CODE REVIEW

**Review by MonikaJhingan — APPROVED** [review]
The UI looks good and code is well indented.It should be merged.

File: src/components/Toast/Toast.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+export const Toast = () => {
+  // const { toast, toastHandler } = useStateContext();
+  // const { showToast, message, type } = toast;
+
+  // useEffect(() => {
+  //   const timeout = setTimeout(() => {
+  //     toastHandler();
+  //   }, 3000);
+  //   return () => clearTimeout(timeout);
+  // }, [showToast]);
```
Comment (shraddha15-creator) [inline src/components/Toast/Toast.js:15]: why so much code is commented? will it required later? if not then remove it

File: src/index.css [hunk 2]
```diff
@@ -1,13 +1,121 @@
...
+  background-color: var(--bg-color);
+  -webkit-border-radius: 10px;
+  border-radius: 10px;
 }
+
+::-webkit-scrollbar-thumb {
+  -webkit-border-radius: 10px;
+  border-radius: 10px;
+  background: var(--font-color-light);
+} */
```
Comment (shraddha15-creator) [inline src/index.css:88]: this as well

**Review by shraddha15-creator — APPROVED** [review]
Code looks clean and readable.
Just remove commented code

File: src/components/Toast/Toast.js (hunk 1 above)
Comment (partha8) [reply src/components/Toast/Toast.js:15]: The commented-out code will be used later. These lines were previously used in my projects so I just copy-pasted them here.

**Comment by partha8** [comment]
thank you @shraddha15-creator and @MonikaJhingan for the reviews.

**Pull request merged**